src/components/committees/committee.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useTranslations } from "next-intl";
+import styles from "@/styles/components/committees/committee.module.scss"
+import Image from 'next/image'
+import Link from 'next/link';
+
+export async function getStaticProps({ locale }) {
+    return {
+      props: {
+        messages: (await import(`@/messages/${locale}.json`)).default
+      }
+    };
+}
+
+
+export default function CommitteeCard({props}) {
+    const t = useTranslations(`CommitteeCards.${props}`)
+    return (
+        <div className={styles.committee_card}>
+            <Link href={t('path')}>
+                <Image
```

Comment:
Also wrap the image with a div that just acts as a container for the image bc that makes responsiveness easier. check out sam's example if you need to.